### PR TITLE
REF: q2-fmt ci

### PR DESCRIPTION
**Pull request by cherman2**
Following dwq2 How-Tos on how to implement CI and GHA for weekly testing

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,25 +8,27 @@
...
-conda env create -n qiime2-fmt-2023.9 --file qiime2-fmt-2023.9-py38-linux-conda.yml
+conda env create \
+ -n q2-fmt-2024.10 \
+ -f https://raw.githubusercontent.com/qiime2/q2-fmt.git/dev/environment-files/2024.10-q2-fmt-environment.yml
 ```
+
 Then activate your new environment as usual.
 ```bash
-conda activate q2-fmt-2023.9
+conda activate q2-fmt
```
Comment (lizgehret) [inline README.md]:
```suggestion
conda activate q2-fmt-2024.10
```

**Review by lizgehret — APPROVED** [review]
overall this lgtm @cherman2! one small inline change to your README for your environment name and this should be ready for merge.

the only thing i noticed (that i'm assuming you did intentionally) is that your env files for 2024.10 and dev are targeting identical channels (2024.10/amplicon/passed) - i don't think it matters, but you could probably consolidate to just the dev env. I don't have strong feelings either way, so you can take that suggestion or leave it as-is!

**Comment by cherman2** [comment]
Thanks @lizgehret! 
You are right that currently those are duplicates. I guess I kinda thought of this as creating a system. Like once .10 is release I will change the env file to use 2024.10/released and I will bump my dev on to use 2025.4/passed. Then I will start making the env files per release like suggested. I would have done a 2024.5 env but q2-fmt wont build there becuase of stats.

Will you do the mergonors (merge honors)?

**Comment by lizgehret** [comment]
Roger that, makes sense to me @cherman2!

lgtm!! 🚀

**Pull request merged**